.github/workflows/auto-comment.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Process Label Action
+        name: Process Label Action
         uses: hramos/respond-to-issue-based-on-label@v2
```

Comment:
There needs to be a hyphen before `uses` on line 14 - see https://github.com/zowe/zowe-client-python-sdk/pull/164/files for an example